### PR TITLE
Add nss_wrapper

### PR DESCRIPTION
**Pull request by chrisburr**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by chrisburr** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/nss_wrapper/build.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -euo pipefail
+IFS=$'\n\t'
+
+mkdir build
+cd build
+cmake -LAH .. \
+    -DCMAKE_INSTALL_LIBDIR=lib \
+    -DCMAKE_INSTALL_PREFIX="${PREFIX}"
```
Comment (marcelotrevisani) [inline recipes/nss_wrapper/build.sh:9]: Should we add the build type to just avoid any problem? :grimacing: 
```suggestion
    -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
    -DCMAKE_BUILD_TYPE=Release
```

File: recipes/nss_wrapper/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - cmake
+    - make
```
Comment (marcelotrevisani) [inline recipes/nss_wrapper/meta.yaml:19]: don't you need ``nss`` to run and compile this?

Comment (chrisburr) [reply recipes/nss_wrapper/meta.yaml:19]: No. It's designed to be used as `LD_PRELOAD=libnss_wrapper.so my_executable` to make it possible to change or edit the `passwd` and `group` files without affecting the host system (or requiring root privileges).

See https://cwrap.org/nss_wrapper.html

**Comment by chrisburr** [comment]
@conda-forge/staged-recipes, ready for review again!

**Comment by chrisburr** [comment]
Thanks @marcelotrevisani! 🍰

**Pull request merged**